packages/design-system/src/styles/components/_SingleInputDateField.scss

```diff
@@ -524,7 +526,7 @@
     text-align: center;
     text-decoration: none;
 
-    &:after {
+    &::after {
```

Comment:
I think the linter rule for having double-colons on `::before` and `::after` is just being pedantic. It's technically correct, but the browser also doesn't care. It's easy enough running `yarn lint:stylelint --fix` to clean these sorts of things up though.

Reply:
I thought at one point single colon had better browser support or something. Agree this is pedantic but neither do I care 😆